src/pages/members.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import DefaultLayout from "../components/layout/defaultLayout";
+import ProfileCard from "../components/shared/profileCard/index";
+const Members = () => {
+    const content = {
```

Comment:
May be create array on and iterate on it ??

Reply:
it's just mock data we need to remove when we develop page